cassandra-scheduler/src/test/resources/scheduler.yml

```diff
@@ -1,17 +1,19 @@
 framework_name : ${FRAMEWORK_NAME:-"cassandra"}
 framework_version : ${FRAMEWORK_VERSION:-"version"}
 executor:
-  command : ${EXECUTOR_COMMAND:-"./executor/bin/cassandra-executor server executor/conf/executor.yml"}
+  command : ${EXECUTOR_COMMAND:-"ulimit -l $CASSANDRA_ULIMIT_MEMLOCK && ulimit -n $CASSANDRA_ULIMIT_NOFILE && ulimit -u $CASSANDRA_ULIMIT_NPROC && ulimit -a && ./executor/bin/cassandra-executor server executor/conf/executor.yml"}
```

Comment:
@mohitsoni @gabrielhartmann 
Isn't this the only line that is needed to accomplish the purpose of the entire PR (and the rest is not necessary)? 
The shell variables are automatically expanded here and the command string should be sent as is to the executor process. All the passing of the variables around from the scheduler to the executor is not needed.